dll/shellext/netplwiz/SHDisconnectNetDrives.c

```diff
@@ -0,0 +1,229 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Disconnect Network Drive
+ * FILE:            dll/shellext/netplwiz/SHDisconnectNetDrives.c
+ * PURPOSE:         Disconnects mapped network drives.
+ * PROGRAMMERS:     Jared Smudde
+ */
```

Comment:
Please use the new file header format: https://www.reactos.org/wiki/Coding_Style#File_Structure